download_and_create_reference_datasets/v03/write_clinvar_ht.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import argparse
+import os
+
+import hail as hl
+
+from hail_scripts.reference_data.constants import GCS_PREFIX
+from hail_scripts.utils.clinvar import (
+    download_and_import_latest_clinvar_vcf,
+    CLINVAR_GOLD_STARS_LOOKUP,
+)
+from hail_scripts.utils.hail_utils import write_ht
+
+CLINVAR_HT_PATH = os.path.join(GCS_PREFIX, 'clinvar/clinvar.GRCh{genome_version}.{timestamp}.ht')
+
+def run(environment: str):
+    for genome_version in ['37', '38']:
+        mt = download_and_import_latest_clinvar_vcf(genome_version)
+        timestamp = hl.eval(mt.version)
+        ht = mt.rows()
+        ht = ht.annotate(
+            gold_stars=CLINVAR_GOLD_STARS_LOOKUP.get(hl.delimit(ht.info.CLNREVSTAT))
+        )
+        ht.describe()
+        ht = ht.transmute(info=ht.info.select('ALLELEID', 'CLNSIG')).select('info', 'gold_stars')
+        ht = ht.repartition(100)
```

Comment:
how is this number chosen? Do we want to compute it more explicitly, or if not should we make it a named constant?

Reply:
This is just copied from the existing script.  I'm not sure!  

I'm down to make this a constant though.

Reply:
Aha: https://github.com/broadinstitute/seqr-loading-pipelines/pull/383